guide/12-enrich-data-with-thematic-information/part3_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    enrich_df.info()
</pre><p>You print the .info() without describing it in many places and I am confused as to why you always have that and then the dataframe. Would it not be enough to just print the dataframe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>